src/app/setup/github-oauth.component.html

```diff
@@ -13,15 +17,16 @@
           <div>
             <h2>Register Syndesis</h2>
             <div>
-              <div *ngIf="!stepOneComplete">
+              <div *ngIf="stepOneComplete === false">
```

Comment:
Would an `*ngSwitch` and a single variable maybe be better for determining what the current step is?

Reply:
Honestly I think we can refactor with a single var later on, but yes that's likely a better approach.